src/test/unit_tests/test_encryption.js

```diff
@@ -813,6 +814,39 @@ mocha.describe('Rotation tests', function() {
                 }
             }));
         });
+
+        [true, false].forEach(update => {
```

Comment:
this test is a bit complicated, where did you check other secrets like chunk cipher keys ?